docs/content/docs/dev/table/materialized-table/overview.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Overview
+weight: 1
+type: docs
+aliases:
+- /dev/table/materialized-table.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Introduction
+
+Materialized tables are a new feature introduced in Flink 1.20 aimed at simplifying both batch and stream processing development processes, providing a consistent development experience. Users only need to specify the query and data freshness when creating a materialized table, and the framework will automatically choose the appropriate execution mode.
+
+{{< hint warning >}}
+This feature is in the MVP stage, only supported for use with `sql gateway` and in `Flink standalone` mode, and is currently not recommended for production use.
```

Comment:
**Note:** This feature is currently an MVP (“minimum viable product”) feature and only available to [SQL Gateway]({{< ref "docs/dev/table/sql-gateway/overview" >}}) and [Standalone]({{< ref "docs/deployment/resource-providers/standalone/overview" >}}) cluster.